mr-clean-reflect/src/main/java/com/trello/mrclean/reflect/Sanitizer.kt

```diff
@@ -0,0 +1,14 @@
+package com.trello.mrclean.reflect
+
+import kotlin.reflect.full.memberProperties
+import kotlin.reflect.jvm.javaGetter
+
+inline fun <reified T: Any> T.sanitizedToString(): String {
+  val instance = this
+  val sanitizedString = instance::class.memberProperties
+      .joinToString {
+        "${it.name} = ${it.javaGetter?.invoke(instance)}"
```

Comment:
So reflect doesn't sanitise anything? Perhaps that should be very clearly outlined in the readme